scene/register_scene_types.cpp

```diff
@@ -1027,6 +1027,26 @@ void register_scene_types() {
 	GDREGISTER_CLASS(NavigationObstacle2D);
 	GDREGISTER_CLASS(NavigationLink2D);
 
+#ifdef MODULE_NAVIGATION_ENABLED
+	OS::get_singleton()->yield(); // may take time to init
```

Comment:
Why do we need a yield here ?

Reply:
It might not be needed but when you look above it is done after some node init bulks because things may have deferred setup time. This may or may not the case, I am not familiar with every single node behavior so this yield is there for safety.